app/views/notes/_notes.html.erb

```diff
@@ -6,5 +6,9 @@
   <span class="govuk-caption-m"><%= note.created_at.to_date.to_formatted_s(:govuk) %></span>
   <h3 class="govuk-heading-m"><%= note.user.full_name || note.user.email %></h3>
 
-  <p class="govuk-body"><%= note.body %></p>
+  <%= sanitize GovukMarkdown.render(note.body) %>
```

Comment:
I think you will need to rebase this work. Could we use the new method I added `render_markdown` here?